docs/operator-guides/aks/troubleshoot-network-aks.md

```diff
@@ -82,11 +82,14 @@ Check the allocated IP addresses in the plugin IPAM store. You might find that a
 
 ```bash
 # Kubenet, for example. The actual path of the IPAM store file depends on network plugin implementation. 
-cd /var/lib/cni/networks/kubenet 
-ls -al|wc -l 
+chroot /host/
+cd /var/lib/cni/networks/k8s-pod-network
```

Comment:
The comment says for "kubenet" -- just to verify, this location is for kubnet?  If not, please update the comment.

Reply:
@ckittel I created two `kubenet` AKS and they are eventually creating this folder. And the folder `kubnet` (which is used by current document) is not existing. 
I did not verify if this is the folder also for `normal CNI`, `CNI overlay`. But I checked it is not for `CNI with Podsubnet`. Or maybe the behaviour will be different if not using custom VNet (although I don't think there will be any). I will further verify later and update the comment.

Reply:
Sounds good.  If you had it on a kubenet cluster, that's good enough for me.  I'll mark it resolved.  If you find something that should require a change with this though, just push it with your next push :)